Dockerfile.PersonsApi

```diff
@@ -0,0 +1,33 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0-bookworm-slim AS build
+WORKDIR /build
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY . .
+
+RUN --mount=type=secret,id=github_token dotnet nuget add source --username USERNAME --password $(cat /run/secrets/github_token) --store-password-in-clear-text --name github "https://nuget.pkg.github.com/DFE-Digital/index.json"
+RUN dotnet build -c Release PersonsApi.sln --no-restore
```

Comment:
I think the solution name is incorrect, it should be TramsDataApi

Reply:
Actually perhaps it needs to be the Project name rather than solution as this is only to publish that specific project!

Reply:
Fixed with [07da94f](https://github.com/DFE-Digital/academies-api/pull/551/commits/07da94f977a3fd7b93976fff0b973ff8c451c027)